src/components/forms/PrivacyPolicy/PrivacyPolicy.js

```diff
@@ -0,0 +1,266 @@
+            signing up.
+          </Card.Text>
+        </Card.Body>
+      </Card>
+    ) : null;
+  };
+
+  return (
+    <Card>
+      <Accordion defaultActiveKey={defaultActiveKey}>
+        <Card.Header>
+          <Accordion.Toggle as="h5" eventKey="0">
+            <Container>
+              <Row>
+                <Col
+                  style={{
+                    display: 'flex',
+                    flexDirection: 'column',
+                    flexBasis: '100%',
+                    flex: 'auto'
+                  }}
+                >
+                  <Button
+                    aria-controls="privacy-policy-button"
+                    disabled={!onSignupPage}
+                  >
+                    Privacy Policy
+                  </Button>
+                </Col>
+                <Col>
+                  <SignupNotice />
+                </Col>
+              </Row>
+            </Container>
+          </Accordion.Toggle>
+        </Card.Header>
+        <Accordion.Collapse eventKey="0">
+          <Card.Body>
+            <ListGroup variant="flush">
+              <ListGroup.Item variant="info">
+                <Card.Text>
+                  We take your privacy seriously. To better protect your privacy
+                  we provide this privacy policy notice explaining the way your
+                  personal information is collected and used. By using the
+                  Service, you agree to the collection and use of information in
+                  accordance with this policy.
+                </Card.Text>
+              </ListGroup.Item>
+              <ListGroup.Item>
+                <Card.Subtitle className="mb-1">
+                  Collection of routine information
+                </Card.Subtitle>
+                <Card.Text>
+                  We collect the routine information in the sign up form above
+                  which includes but is not limited to (name, email and contact
+                  number).
+                </Card.Text>
+                <Card.Text>
+                  We use this information to operate, maintain, and provide to
+                  you the features and functionality of the Service. We do not
+                  sell, trade or otherwise transfer this information with any
+                  external third party organisations, however, in signing up it
+                  will be shared with the City Harvest Volunteer coordinator, as
+                  it is required for them to approve you as a new Volunteer. We
+                  also use this data to allow the Volunteer coordinator to
+                  contact you in case of any emergencies or potential shifts and
+                  you will be able to select your contact preferences after
+                  being accepted by the coordinator.
+                </Card.Text>
+                <Card.Text>
+                  Please note: The Charity may also ask additional questions as
```

Comment:
As we aren't doing additional questions right now, this section may not need to be here